OMEdit/OMEditLIB/Annotations/BitmapAnnotation.cpp

```diff
@@ -209,11 +249,10 @@ void BitmapAnnotation::paint(QPainter *painter, const QStyleOptionGraphicsItem *
 void BitmapAnnotation::drawBitmapAnnotation(QPainter *painter)
 {
   QRectF rect = getBoundingRect().normalized();
-  QImage image = mImage.scaled(rect.width(), rect.height(), Qt::KeepAspectRatio, Qt::SmoothTransformation);
-  QPointF centerPoint = rect.center() - image.rect().center();
-  QRectF target(centerPoint.x(), centerPoint.y(), image.width(), image.height());
 
-  painter->drawImage(target, mImage.mirrored());
```

Comment:
FYI: This is the root place where the change actually needed to be applied at. Instead of always drawing image (bitmap), we now delegate the painting operation to whatever the `mpRenderer` runtime type has been constructed. Bitmap images render pretty much the same way (see `BitmapRenderer` class) and SVG images use `QSvgRenderer` under the hood instead (see `SvgRenderer class).